<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameFunc</class>
 <widget class="QFrame" name="FrameFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrameFunc</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>方法选择</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_bisection">
          <property name="text">
           <string>二分法</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_newdon">
          <property name="text">
           <string>牛顿迭代法</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="FrameDraw" name="wdgPaint"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>输出信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>当前迭代数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_curitcnt">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>当前迭代结果X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_curitcnt_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>输入</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>求解方程：f(x)=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_funcExpr"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>迭代式  φ (x)=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_itExpr"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>初值X0        :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_x0">
            <property name="inputMask">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="edt_other">
            <property name="text">
             <string>特殊输入 :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>a1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_a1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>a2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_a2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>XX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_a2_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>精度要求</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_precision">
            <item>
             <property name="text">
              <string>0.0001</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.000001</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.00000001</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>操作</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_read">
          <property name="text">
           <string>准备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>迭代一次</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameDraw</class>
   <extends>QFrame</extends>
   <header>FrameDraw/FrameDraw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
